go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/hashicorp/terraform-exec v0.20.0
 	github.com/hashicorp/terraform-json v0.21.0
 	github.com/hashicorp/terraform-registry-address v0.2.3
-	github.com/hashicorp/terraform-schema v0.0.0-20240326154150-560df6bbc4dc
```

Comment:
Since the last release of `terraform-schema` happened 8 months ago, I suppose we can just update this to the (rebased) commit on `main`, right? (I just merged the other PR that this PR depends on)

Reply:
updated to `364ac1` which is the current `main` (after the "depends on"-PR named above had been merged)

Reply:
Yes, that is the process we usually follow:
- Merge the dependent PR in terraform-schema
- Rebase this PR with the new commit hash from terraform-schema's `main` branch

To make the test pass during review, we first pin it to the PR commit